src/components/ProviderContainer.tsx

```diff
@@ -105,6 +112,11 @@ const SBComponent = ({ children }: { children: React.ReactElement }) => {
                 },
                 enableVoiceMessage: false,
                 enableFeedback: enableEmojiFeedback,
+                /**
+                 * Please keep below comments for easy local testing.
+                 */
+                // enableSuggestedReplies: true,
+                // showSuggestedRepliesFor: 'all_messages',
```

Comment:
We're using a custom suggested reply component in Widget at the moment, so this won't work 😅

Reply:
removed it!